composer.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "antidot-be/bpost-api-library",
+  "name": "yentel/bpost-api-library",
   "version": "3.4.0",
   "type": "library",
   "description": "bpost API library is a PHP library to communicate with the bpost API.",
-  "homepage": "https://github.com/Antidot-be/bpost-api-library",
+  "homepage": "https://github.com/Yentel/bpost-api-library",
```

Comment:
?

Reply:
Hmmmm, that second commit shouldn't have been part of this pull-request
I'll fix this tomorrow, sorry! I added this change to require this fork in our own projects until the merge was accepted so we could change back to antidot afterwards. My bad! ;-)

Reply:
No worries :-)